src/jobs_lib.erl

```diff
@@ -33,3 +35,10 @@ timestamp_to_datetime(TS) ->
     MS = TS rem 1000,
     %% return {Datetime, Milliseconds}
     {calendar:now_to_datetime({1258,S,0}), MS}.
+
+
+compat_now() ->
+  try erlang:timestamp()
```

Comment:
If called on an older erlang version, this will result in a call to `code_server`. Better to check using `erlang:is_builtin(erlang, timestamp, 0)`. Example, using R16B03:

```erlang
5> try erlang:timestamp() catch error:undef -> undefined end.
(<0.41.0>) call code_server:call(code_server,{ensure_loaded,erlang})
(<0.41.0>) returned from code_server:call/2 -> {module,erlang}
undefined
6> erlang:is_builtin(erlang,timestamp,0).
false
7> erlang:is_builtin(erlang,now,0).      
true
```